app/views/content_items/fatality_notice.html.erb

```diff
@@ -14,15 +14,15 @@
 <div class="grid-row">
   <div class="column-two-thirds content-bottom-margin">
     <div class="responsive-bottom-margin">
+      <%= render 'components/important-metadata',
+          items: @content_item.metadata[:other] %>
       <%= render 'components/figure',
           src: @content_item.image["url"],
           alt: @content_item.image["alt_text"],
           caption: @content_item.image["caption"] if @content_item.image %>
-      <%= render 'components/important-metadata',
-          items: @content_item.metadata[:other] %>
       <%= render 'govuk_component/govspeak',
-        content: @content_item.body,
-        direction: page_text_direction %>
+          content: @content_item.body,
```

Comment:
Indentation looks different to the rest of this template, I don't mind which way it goes, so long as linting is happy and it's consistent.